readme.md

```diff
@@ -144,6 +144,45 @@ const emitter = require('./some-event-emitter');
 })();
 ```
 
+### pEvent.multiple(emitter, event, options)
+
+With `pEvent.multiple()` there comes a few new options:
+
+##### count
+
+Type: `number`
+
+Tells how many times the event needs to be emitted.
+
+##### resolveImmediately
+
+Type: `boolean`<br>
+Default: `false`
+
+States if the promise should resolve immediately. Emitting one of the `rejectionEvents` won't throw an error. Example:
```

Comment:
This should provide a use-case for the option, as it's not immediately clear, and also clarify that the returned array will be mutated. when new events are emitted.